### DISCUSSION TITLE
Gotta love Frutiger Aero♥

### DISCUSSION THREAD

**Discussion by TheInsane101** [Maplespe/DWMBlurGlass#184] [Show and tell]
![Screenshot (957)](https://github.com/Maplespe/DWMBlurGlass/assets/104862146/8cf6391e-1d1d-4251-af43-9ffcad5fb4e5)

**Comment by OldManAlpha** [comment]
What did you do to get the old tray icons?

**Reply by TheInsane101** [reply]
The icon pack that came in this https://www.deviantart.com/windowz3e/art/Win10-to-Aero-7-Vista-Longhorn-965218718

**Reply by OldManAlpha** [reply]
Thank you, that icon pack changes a lot more icons than I expected!
